bears/go/GoMetaLinterBear.py

```diff
@@ -0,0 +1,33 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='gometalinter.v1',
+        use_stderr=True,
+        output_format='regex',
+        output_regex=r'(.*):(\d*):(\d*):(.*) \((\w*)\)')
+class GoMetaLinterBear:
+    """
+    Checks the code using ``gometalinter``. This will run
+    some set of golang linters over each file separately.
+    """
+    LANGUAGES = {'Go'}
+    REQUIREMENTS = {GoRequirement(
+        package='gopkg.in/alecthomas/gometalinter.v1', flag='-u')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting','Syntax','Missing import',
+                  'Unused Code', 'Smell', 'Unreachable Code','Security'}
+    SEE_MORE = 'https://github.com/alecthomas/gometalinter'
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
```

Comment:
maybe you should make it configurable by enabling or disabling some linters ? :sweat_smile: